client/js/components/statement/segments/DpInsertableRecommendation.vue

```diff
@@ -1,62 +1,77 @@
 <license>
-  (c) 2010-present DEMOS plan GmbH.
```

Comment:
this file can be removed from core, if i get this correctly?

Reply:
Like the addon wrapper we decided to keep this file in the core and copy it to places where we need them